src/Polymorph-Widgets/UITheme.class.st

```diff
@@ -5321,6 +5321,13 @@ UITheme >> selectedFillStyleFor: aMorph height: anInteger [
 
 ]
 
+{ #category : #'accessing colors' }
+UITheme >> selectedItemListTextColor [
+	"Answer the color for the selected item in a list text."
+
+	^ Color black
```

Comment:
default selected color needs to be "textColor", not black.

Reply:
Ok I will check if it is working with my style. My problem is that I could not see the text of certain items. 
Thanks for the feedback

Reply:
This does not work for me. 


![Screenshot 2020-02-01 at 11 33 58](https://user-images.githubusercontent.com/432549/73590786-c2d30e80-44e6-11ea-8a87-b95a3819b324.png)

Reply:
This is the only place in the system it does not work. 
I do not understand why this is happening. In particular since

In my theme I redefined textColor this way. 

	"This is only for all the list and the text except that the selection color can hide it on certain list."
	^ Color black

Reply:
I'm investigating (but my brain is fucked by the flu) still I spent a lot of time on this issue in the past. There is something strange.

Reply:
Now I'm totally confused :( So probably should go to bed. ;(

Reply:
Ok apparently lazyListMorph is used in dropBox. Not the finder tree.

Reply:
So for the dropbox what you propose is working.